app/main/views/briefs.py

```diff
@@ -360,15 +360,21 @@ def view_response_result(brief_id):
         return redirect(url_for(".brief_response", brief_id=brief_id))
     elif brief_response[0].get('essentialRequirementsMet') or all(brief_response[0]['essentialRequirements']):
         result_state = 'submitted_ok'
+        flash('submitted_ok', 'success')
```

Comment:
Does this mean we set a flash message for success everytime you visit the page and have submitted succesfully? Or should this just be shown the first time you complete the flow, i.e. the flash is set on the penultimate page?